src/mono/wasm/runtime/startup.ts

```diff
@@ -49,6 +50,22 @@ export function configure_emscripten_startup(module: DotnetModule, exportedAPI:
         }
     }
 
+    console.debug ("MONO_WASM: Initialize WebWorkers");
+    // Consider the can_call_digest check in the pal_crypto_webworker when updating this check.
+    if (ENVIRONMENT_IS_WEB && typeof crypto.subtle === "undefined" && typeof SharedArrayBuffer !== "undefined") {
+        const chan = LibraryChannel.create(1024); // 1024 is the buffer size in char units.
+        const worker = new Worker("dotnet-crypto-worker.js");
+        (globalThis as any).mono_wasm_crypto = {
```

Comment:
This is polluting global namespace. Please rather add it to `INTERNAL` in `exports.ts`

Reply:
> This is polluting global namespace. Please rather add it to `INTERNAL` in `exports.ts`

I think there were scoping issues that might need your help to resolve.